app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Api::V1::UsersController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def create
+    if body_params[:username]
```

Comment:
We test it and the rails return an HTML rails error and we already verify if the value is empty and return it as a JSON answer, so there is no need to use require here since it breaks the code.